pages/views.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.views import APIView
+from pages.models import Term
+from pages.serializers import PagesSerializer
+from rest_framework.response import Response
+from rest_framework import status
+
+
+class TermsView(APIView):
+    """Handles sending terms and conditions to users."""
+    serializer_class = PagesSerializer
+
+    def get(self, request):
+        terms = Term.objects.latest('last_updated_at')
```

Comment:
Hey @CryceTruly , great work, could you kindly add a docstring here describing what this get method does.

Reply:
Thanks, @Genza999, I have worked on this.